handler.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const uuid = require('uuid');
+const AWS = require('aws-sdk'); 
+AWS.config.setPromisesDependency(require('bluebird'));
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+/* POST /guests */
+module.exports.submitGuest = async (event) => {
+	const requestBody = JSON.parse(event.body);
+  const firstName = requestBody.firstName;
+  const lastName = requestBody.lastName;
+  const responded = requestBody.responded;
+
+	if (typeof firstName !== 'string' || typeof lastName !== 'string' || typeof responded !== 'boolean') {
+    return {
+      statusCode: 400,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Methods': 'OPTIONS, POST, GET',
+      },
+      body: JSON.stringify({
+        message: `Validation failed. Invalid input type(s).`,
+      })
+    }
+  }
+
+	const guest = guestItem(firstName, lastName, responded);
+
+  await putGuest(guest);  
+  return {
+    statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Methods': 'OPTIONS, POST, GET',
+    },
+    body: JSON.stringify({
+      message: `Added ${firstName} ${lastName}`,
+    })
+  }
+}
+
+/* Puts guest in dynamo table */
+const putGuest = async (guest) => {
+  const guestItem = {
+    TableName: process.env.GUESTS_TABLE,
+    Item: guest,
+  };
+  
+    await dynamoDb.put(guestItem).promise();
+    return guest;
+}
+
+/* Creates a guest item */
+const guestItem = (firstName, lastName, responded) => {
```

Comment:
This is basically a "builder" pattern for object creation. Let's rename this to be more descriptive about what it's doing:
```suggestion
const buildGuestItem = function buildGuestDynamoModel(firstName, lastName, responded) {
```
Also we should prefer using named function expressions like in the Airbnb JavaScript style guide: https://github.com/airbnb/javascript#functions

It makes it easier to debug since the stack trace will have the named function instead of an anonymous one.